ci-operator/step-registry/workers-rhel/pre-hook/azure/workers-rhel-pre-hook-azure-commands.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+# Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+# to be able to SSH.
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
+
+cat > scaleup-pre-hook-azure.yaml <<- 'EOF'
+- name: configuraiton RHEL machine on Azure
+  hosts: new_workers
+  any_errors_fatal: true
+  gather_facts: false
+  tasks:
+  - name: Extend the lv of /var for RHEL machine
+    shell: |
+      sudo lvextend -L+20G /dev/rootvg/varlv
```

Comment:
Let us replace `lvextend -L+20G` with `lvextend -l +100%FREE` to make it a bit wiser.

Reply:
That means there are some free disk spaces not allocated yet, if we want to use the full size of disk, need to create a new partition, use it to create a pv, then add the pv to vg, then lvextend again. Per our QE past testing in jenins ci pipleine, seem like 49G /var is enough for testing. So we do not need to set 128G disk for now, setting it to 80G by default for `RHEL_VM_DISK_SIZE` should be enough in workers-rhel-azure-provision-ref.yaml, then drop other places `RHEL_VM_DISK_SIZE` setting.

Reply:
agree, I saw that size of disk /var is 20G on vsphere rhel worker, and seems not hit disk pressure issue so far. I updated to set 64G by default in provision ref, which is same value as in `v4-sacleup` repo.